src/sql/sqlops.proposed.d.ts

```diff
@@ -676,4 +676,11 @@ declare module 'sqlops' {
 			openEditor(position?: vscode.ViewColumn): Thenable<void>;
 		}
 	}
+
+	export interface ModelViewEditorOptions {
```

Comment:
It might be better to put this interface in the `sqlops.workspace` namespace since the only time it's used is in `sqlops.workspace.createModelViewEditor`